src/charts/tooltip.js

```diff
@@ -7,6 +7,8 @@ define(function(require){
     const d3Transition = require('d3-transition');
     const d3TimeFormat = require('d3-time-format');
 
+    const NUMBER_FORMAT = null;
```

Comment:
Instead of having a constant for it, I would directly assign `null` in line `116`